README.md

```diff
@@ -3,93 +3,89 @@
-    * [Turning on Network Inspection](#turning-on-network-inspection)
-    * [Turning on Headless Mode](#turning-on-headless-mode)
-  * [Using the tools](#using-the-tools)
-    * [Setting up your project to show live changes in the extension](#setting-up-your-project-to-show-live-changes-in-the-extension)
-    * [Opening source files from the Elements tool](#opening-source-files-from-the-elements-tool)
-    * [Debug Configuration](#debug-configuration)
-    * [Other optional launch config fields](#other-optional-launch-config-fields)
-    * [Sourcemaps](#sourcemaps)
-    * [Ionic and gulp sourcemaps note](#ionic-and-gulp-sourcemaps-note)
+* [Getting Started](#getting-started)
+* [Using the tools](#using-the-tools)
   * [Launching the browser via the side bar view](#launching-the-browser-via-the-side-bar-view)
-  * [Launching the browser manually](#launching-the-browser-manually)
+  * [Opening source files from the Elements tool](#opening-source-files-from-the-elements-tool)
+  * [Getting Console information](#getting-console-information)
+  * [Setting up your project to show live changes in the extension](#setting-up-your-project-to-show-live-changes-in-the-extension)
+* [Changing Extension Settings](#changing-extension-settings)
+  * [Turning on Network Inspection](#turning-on-network-inspection)
+  * [Turning on Headless Mode](#turning-on-headless-mode)
+* [Debug Configuration](#debug-configuration)
+  * [Other optional launch config fields](#other-optional-launch-config-fields)
   * [Attaching automatically when launching the browser for debugging](#attaching-automatically-when-launching-the-browser-for-debugging)
 * [Contributing](#contributing)
 * [Other information](#other-information)
-  * [Data and Telemetry](#data-telemetry)
-  * [Reporting Security Issues](#reporting-security-issues)
+* [Data and Telemetry](#data-and-telemetry)
+* [Reporting Security Issues](#reporting-security-issues)
+
 
 ## Supported Features
 
-* Debug configurations for launching Microsoft Edge browser in remote-debugging mode and auto attaching the tools.
-* Side Bar view for listing all the debuggable targets, including tabs, extensions, service workers, etc.
 * Fully featured Elements and Network tool with views for HTML, CSS, accessibility and more.
 * Screen-casting feature to allow you to see your page without leaving Visual Studio Code.
-* Go directly to the line/column for source files in your workspace when clicking on a link or CSS rule inside the Elements tool.
-* Auto attach the Microsoft Edge Tools when you start debugging with the [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) extension.
+* Debug configurations for launching Microsoft Edge browser in remote-debugging mode and auto attaching the tools.
 * Debug using a windowed or headless version of the Microsoft Edge Browser
+* Side Bar view for listing all the debuggable targets, including tabs, extensions, service workers, etc.
```

Comment:
As a note, service and shared workers are hidden from the target list by default and can be enabled via settings.  Up to you whether we want to publicize a disabled-by-default functionality.

Reply:
Let's not for now, and maybe think about how we could make that more interesting. As we don't have the applications tab for debugging service workers it seems superfluous.